applications/luci-app-ddns/root/usr/libexec/rpcd/luci.ddns

```diff
@@ -109,15 +109,15 @@ local methods = {
 					tonumber(s["force_interval"]) or 72,
 					s["force_unit"] or "hours" )
 
-				-- process running but update needs to happen
-				-- problems if force_seconds > uptime
-				force_seconds = (force_seconds > uptime) and uptime or force_seconds
+				local check_seconds = calc_seconds(
+					tonumber(s["check_interval"]) or 10,
+					s["check_unit"] or "minutes" )
 
 				if last_update > 0 then
-					local epoch = os.time() - uptime + last_update + force_seconds
+					local epoch = os.time() - uptime + last_update
 					-- use linux date to convert epoch
 					converted_last_update = epoch2date(epoch,date_format)
-					next_update = epoch2date(epoch + force_seconds)
+					next_update = epoch2date(epoch + force_seconds + check_seconds)
```

Comment:
@small-5 I would say it fits.
@Ansuel I just noticed, why didn't we also specify date_format as the second argument here?

Reply:
I need to check the script